sdcm/cdclog_reader_thread.py

```diff
@@ -0,0 +1,198 @@
+import time
+import logging
+import uuid
+import pprint
+
+from pathlib import Path
+from typing import List, Dict
+
+from sdcm.sct_events import CDCReaderStressEvent, Severity
+from sdcm.utils.common import FileFollowerThread, get_docker_stress_image_name
+from sdcm.utils.thread import DockerBasedStressThread
+from sdcm.utils.docker import RemoteDocker
+from sdcm.stress_thread import format_stress_cmd_error
+
+LOGGER = logging.getLogger(__name__)
+
+CDC_LOGTABLE_SUFFIX = "_scylla_cdc_log"
+CDCLOG_READER_IMAGE = get_docker_stress_image_name(tool_name="cdcstressor")
+PP = pprint.PrettyPrinter(indent=2)
+
+
+class CDCLogReaderStatsPublisher(FileFollowerThread):
+    def __init__(self, loader_node, loader_idx, cdclogreader_filename):
+        super().__init__()
+        self.loader_node = loader_node
+        self.loader_idx = loader_idx
```

Comment:
those are not really used, if this class gonna stay

Reply:
removed